dpctl-capi/source/dpctl_sycl_device_interface.cpp

```diff
@@ -387,3 +389,89 @@ bool DPCTLDevice_HasAspect(__dpctl_keep const DPCTLSyclDeviceRef DRef,
     }
     return hasAspect;
 }
+
+__dpctl_give DPCTLDeviceVectorRef
+DPCTLDevice_CreateSubDevicesEqually(__dpctl_keep const DPCTLSyclDeviceRef DRef,
+                                    size_t count)
+{
+    vector_class<DPCTLSyclDeviceRef> *Devices = nullptr;
```

Comment:
We can use the opaque type `DPCTLDeviceVectorRef` instead of `vector_class<DPCTLSyclDeviceRef>`, *e.g,* `DPCTLDeviceVectorRef Devices = nullptr`.